tests/console/python_flake8.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# Summary: Regression test for python-flake8
+# Maintainer: Avinesh Kumar <akumar@suse.de>
+
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # make sure that python-flake8 is installed
+    zypper_call 'in python3-flake8';
+
```

Comment:
Don't assume that the package hub extension is already installed or at least leave a comment about it, to make sure that in the future we know what's needed for it to work...

Reply:
Added a comment mentioning phub extension is required for this package to be installed on SLE.